### PR TITLE
chore(release): 2.0.0

### PR DESCRIPTION
**Pull request by rsmayda**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.0.0](https://github.com/awslabs/fhir-works-on-aws-authz-smart/compare/v1.0.1...v2.0.0) (2021-06-24)
+
+
+### ⚠ BREAKING CHANGES
+
+* Allow paths for fhirUser and launch claims (#40)
```
Comment (carvantes) [inline CHANGELOG.md]: I think that the breaking change description is not specific enough. We want to let customers know exactly what needs to be changed on their side to upgrade to this release.

btw, I think that it is almost always better to include `BREAKING CHANGE: <change details>` on commit msgs. `feat!` alone is often not specific enough.

A good commit message for https://github.com/awslabs/fhir-works-on-aws-authz-smart/commit/332806dbeff80c929f36b01c699cf0dac65cea01 could be:
```
feat!: Allow paths for fhirUser and launch claims (#40) 

BREAKING CHANGE: Renamed `fhirUserClaimKey` to `fhirUserClaimPath` and `launchContextKeyPrefix` to `launchContextPathPrefix` in `SMARTConfig`
```

We can modify the CHANGELOG manually this time.

https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer

**Review by carvantes — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (rsmayda) [reply CHANGELOG.md]: Great feedback

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,20 @@
...
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.0.0](https://github.com/awslabs/fhir-works-on-aws-authz-smart/compare/v1.0.1...v2.0.0) (2021-06-24)
+
+### ⚠ BREAKING CHANGES
+
+- Allow paths for fhirUser and launch claims (#40)
+  - Renamed `fhirUserClaimKey` to `fhirUserClaimPath` and `launchContextKeyPrefix` to `launchContextPathPrefix` in `SMARTConfig`
+- Add support for `system` scope (#41)
+  - You must now define how you want to handle the `system` scope found in ScopeRule
```
Comment (carvantes) [inline CHANGELOG.md]: nit:

```suggestion
* Renamed `fhirUserClaimKey` to `fhirUserClaimPath` and `launchContextKeyPrefix` to `launchContextPathPrefix` in `SMARTConfig`
* You must now define how you want to handle the `system` scope found in ScopeRule
```

I feel like adding the commit msg here is redundant.
Also, a simple list with only the breaking change description is the format that `standard-version` would output

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,36 +24,34 @@ All notable changes to this project will be documented in this file. See [standa
 
 ### [0.1.3](https://github.com/awslabs/fhir-works-on-aws-authz-smart/compare/v0.1.2...v0.1.3) (2021-02-11)
 
-
 ### Features
 
-* make reference checking more restrictive and abide by FHIR ([#22](https://github.com/awslabs/fhir-works-on-aws-authz-smart/issues/22)) ([ca3d574](https://github.com/awslabs/fhir-works-on-aws-authz-smart/commit/ca3d5740bf1d229eb92ec4f5795dede96d1671ae))
-* make smartHandler more configurable ([#24](https://github.com/awslabs/fhir-works-on-aws-authz-smart/issues/24)) ([503ef8a](https://github.com/awslabs/fhir-works-on-aws-authz-smart/commit/503ef8a3a34ad2b4e65765292273e341cb493b0a))
+- make reference checking more restrictive and abide by FHIR ([#22](https://github.com/awslabs/fhir-works-on-aws-authz-smart/issues/22)) ([ca3d574](https://github.com/awslabs/fhir-works-on-aws-authz-smart/commit/ca3d5740bf1d229eb92ec4f5795dede96d1671ae))
```
Comment (carvantes) [inline CHANGELOG.md:27]: nit: not sure why we are changing from `*` to `-` for lists. afaik they do exactly the same, so nothing changes visually

Also, the `standard-version` tool uses `*` anyways, so I think it'd be easier to just use `*` everywhere.

**Review by carvantes — DISMISSED** [review]

**Review by Bingjiling — APPROVED** [review]
LGTM

**Pull request merged**